dragonchain/job_processor/job_processor.py

```diff
@@ -32,7 +32,7 @@
 INTERNAL_ID = os.environ["INTERNAL_ID"]
 STAGE = os.environ["STAGE"]
 REGISTRY = os.environ["REGISTRY"]
-IAM_ROLE = os.environ["IAM_ROLE"]
+IAM_ROLE = os.environ.get("IAM_ROLE") or "N/A"
```

Comment:
I'm all for the `.get`, but not sure why we would hardcode `N/A` there. Why not just have no annotations if the environment variable doesn't exist?

Reply:
agreed. fixed